proto/transparent_operator_stream.proto

```
@@ -0,0 +1,58 @@
+// serialized OperatorMessage, and closes it with FIN — the stream boundary frames the message, so
+// there is NO length prefix and one stream carries exactly one OperatorMessage.
+//
+// Direction:
+//   rtsp-server -> GUI : StatusUpdate   (device-reported status / BAF-1651 response envelope)
+//   GUI -> rtsp-server  : CommandRequest (a BAF-1651 command envelope)
+//
+// The BAF-1651 JSON envelope ({type, version, action, id, timestamp, payload}) is passed through
+// VERBATIM as opaque bytes — this protocol only carries the Fleet Protocol device_id + timestamp
+// needed to route the message, and stays decoupled from the BAF-1651 schema.
+// NB: `operator_stream`, not `operator` — `operator` is a C++ keyword (invalid generated namespace).
+package bringauto.transparent.operator_stream;
```

Buf (1.71.0)

[error] Files with package "bringauto.transparent.operator_stream" must be within a directory "bringauto/transparent/operator_stream" relative to root but were in directory "proto".

(PACKAGE_DIRECTORY_MATCH)